docs/TranslatableInputs.md

```diff
@@ -49,6 +49,7 @@ You may have inputs which are translated in multiple languages and want users to
 | `defaultLocale` | Optional | `string` | `en` | The default locale to display |
 | `groupKey` | Optional | `string` | - | A unique key for accessibiliyty purpose |
 | `selector`| Optional | `ReactNode` | - | A selector to choose the locale to display |
+| `StackProps`| Optional | `object` | - | Props passed to the rendered MUI Stack |
```

Comment:
could you also add the `fullWidth` and `sx` props? Or explain that it accepts the generic input props?